src/controllers/CinemaController.java

```diff
@@ -7,90 +7,91 @@
 import src.entities.*;
 
 public class CinemaController {
-    private String filename;
-    private String seatFileName;
+	private String filename;
+	private String seatFileName;
 
-    public CinemaController(String filename, String seatFileName) {
-	this.filename = filename;
-	this.seatFileName = seatFileName;
-    }
-
-   public void insertCinemaIntoDB(String cinemaName, ArrayList<Movie> movies, HashMap<Movie, ArrayList<LocalDate>> showtimes) {
-	HashMap<String, ArrayList<Seat>> seats = this.getSeatsFromDB(movies, showtimes);
-	ArrayList<Cinema> cinemas = new ArrayList<>();
-	Cinema newCinema = new Cinema(cinemaName, movies, showtimes, seats);
-	FileOutputStream fos = null;
-	ObjectOutputStream out = null;
-	File f = new File(filename);
-	if(f.exists())
-	cinemas = this.getAllCinemasFromDB();//Read in existing data in db
-	else {
-		System.out.println("File: " + filename + " does not exist");
-		System.out.println("Creating new DB");
+	public CinemaController(String filename, String seatFileName) {
+		this.filename = filename;
+		this.seatFileName = seatFileName;
 	}
-	cinemas.add(newCinema);
-	try {
-		fos = new FileOutputStream(filename);
-		out = new ObjectOutputStream(fos);
-		out.writeObject(cinemas);
-		out.close();
-	} catch (IOException ex) {
-		ex.printStackTrace();
+
+	public void insertCinemaIntoDB(String cinemaName, ArrayList<Movie> movies, HashMap<String, ArrayList<LocalDate>> showtimes) {
```

Comment:
Renamed the HashMap<**Movie**, ArrayList<LocalDate>> to HashMap<**
**, ArrayList<LocalDate>>